libraries/botbuilder-dialogs-adaptive/etc/botbuilder-dialogs-adaptive.api.md

```diff
@@ -1071,10 +1071,11 @@ export class OnCancelDialog extends OnDialogEvent {
 
 // @public
 export class OnChooseEntity extends OnDialogEvent {
-    constructor(property?: string, entity?: string, actions?: Dialog[], condition?: string);
+    constructor(property?: string, value?: string, operation? string, actions?: Dialog[], condition?: string);

```

Comment:
It would be potentially useful to add something to CI that regenerates these files and makes sure there are no diffs (to make sure we keep them up to date).